specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -4138,175 +4319,9 @@
             "$ref": "#/parameters/SubscriptionId"
           },
           {
-            "name": "intentObjectName",
+            "name": "fabricName",
             "in": "path",
-            "description": "Replication protection intent name.",
-            "required": true,
-            "type": "string"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/ReplicationProtectionIntent"
-            }
-          }
-        },
-        "x-ms-examples": {
-          "Gets the details of a Replication protection intent item.": {
-            "$ref": "./examples/ReplicationProtectionIntents_Get.json"
-          }
-        }
-      },
-      "put": {
-        "tags": [
-          "ReplicationProtectionIntent"
-        ],
-        "summary": "Create protection intent Resource.",
-        "description": "The operation to create an ASR replication protection intent item.",
-        "operationId": "ReplicationProtectionIntents_Create",
-        "consumes": [
-          "application/json"
-        ],
-        "produces": [
-          "application/json"
-        ],
-        "parameters": [
-          {
-            "$ref": "#/parameters/ApiVersion"
-          },
-          {
-            "$ref": "#/parameters/ResourceName"
-          },
-          {
-            "$ref": "#/parameters/ResourceGroupName"
-          },
-          {
-            "$ref": "#/parameters/SubscriptionId"
-          },
-          {
-            "name": "intentObjectName",
-            "in": "path",
-            "description": "A name for the replication protection item.",
-            "required": true,
-            "type": "string"
-          },
-          {
-            "name": "input",
-            "in": "body",
-            "description": "Create Protection Intent Input.",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/CreateProtectionIntentInput"
-            }
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/ReplicationProtectionIntent"
-            }
-          }
-        },
-        "x-ms-examples": {
-          "Create protection intent Resource.": {
-            "$ref": "./examples/ReplicationProtectionIntents_Create.json"
-          }
-        }
-      }
-    },
-    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{resourceName}/replicationFabrics/{fabricName}/replicationProtectionContainers/{protectionContainerName}/replicationProtectionContainerMappings/{mappingName}": {
-      "get": {
```

Comment:
Same above, as I know, there is a review process for the deprecated API.

Reply:
I am seeing this API being present in swagger doc.
can you please check. It might have been added at some other location.
@lirenhe

Reply:
@lirenhe : Any updates on this?

Reply:
@lirenhe : Can you please check this..
I have updated all review comments.